### ISSUE TITLE
Feature Request: Show a function's documentation reference when a dev clicks a functions `@inheritdoc` tag

### ISSUE THREAD

**Issue by scorpion9979** [NomicFoundation/hardhat-vscode#133]
I noticed that the extension currently has no support for the `@inheritdoc` tag, and I think it would be a very useful feature to be able to display an overridden function's interface documentation in the VSCode references.

![image](https://user-images.githubusercontent.com/4002635/159511408-d30abfcd-f984-4bf7-a274-50022b032b53.png)

![image](https://user-images.githubusercontent.com/4002635/159512788-9e3b96dd-5b2c-452c-b70a-1685a11ff781.png)
Note: code above is from [hifi-finance/hifi](https://github.com/hifi-finance/hifi).

**Comment by kanej** [comment]
Hey @scorpion9979, just to clarify is the idea here that the inheritdoc would become a link of some sort, or that we swap in the documentation inline?

**Comment by scorpion9979** [comment]
> Hey @scorpion9979, just to clarify is the idea here that the inheritdoc would become a link of some sort, or that we swap in the documentation inline?

Yes, the first part is closer to what I have in mind. I think it would be cool to make the inheritdoc tag link to the smart contract's inline documentation of that particular function.
